### DISCUSSION TITLE
Stash - add option to hide window when cursor leaves the window UNLESS holding command

### DISCUSSION THREAD

**Discussion by stuart-leitch** [rxhanson/RectanglePro-Community#126] [Ideas]
Stash is awesome, so good for getting clutter off my desktop, but having it rapidly available.  I really like the feature to re-hide when the cursor leaves the stashed window, and have that as my default.  Occasionally I want to be able to switch into another window without the stashed one sliding gracefully aside.  In that case, I'd like to be able to hold cmd as I jump the border and have the stashed window remain visible.  I can then manually re-hide it in the usual way when I've done whatever I needed to.

Thanks for a genuinely game-changing tool.  I've always used everything full-screen on my mac built in display, but now when working with a nice big external screen Rectangle (Pro!) is invaluable.

**Comment by stuart-leitch** [comment]
Another wee stash request...

Stash and mission control kinda collide.  By that I mean that MC shows the stashed (and hidden) window, but if you click it doesn't reappear.  Could you minimise the stashed windows when they are hidden so they don't show up in MC?  That means that MC would just show the unstashed windows.

**Reply by rxhanson** [reply]
Unfortunately, I don't have fine-grain control over un-minimizing & associated timing to make it smooth enough to tie into unstashing. Seems like disabling the genie effect would be desirable for this as well? I'll give it some more thought once I'm back in the Stash code.

**Comment by rxhanson** [comment]
This makes sense to me, so I'll add it to my list. I have a couple of other Stash related items to implement, so it's a matter of Stash-related work getting priority again.
